File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/EarnedReward.java

package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EarnedReward {

    @SerializedName("rewardId")
    @Expose
    private String rewardId;
    @SerializedName("selected")
    @Expose
    private boolean selected;

    public String getRewardId() {
        return rewardId;
    }

    public void setRewardId(String rewardId) {
        this.rewardId = rewardId;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/EarnedReward.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Earned reward.
 */
@Keep
data class EarnedReward(

    @SerializedName("rewardId")
    var rewardId: String,

    @SerializedName("achievementId")
    var achievementId: String,

    @SerializedName("selected")
    var isSelected: Boolean

)